docker/docker-dev/datasources/graphite

```diff
@@ -1,7 +1,7 @@
 {
   "name":"graphite",
   "type":"graphite",
-  "url":"http://graphite",
+  "url":"http://graphite:80",
```

Comment:
why do we need this?

Reply:
I updated docker-dev to match docker-standard

Reply:
docker-dev has always worked fine for me without the explicit port, i recommend you take out this change out of the commit. it seems like setting the example to others in a confusing way (so we should probably simplify docker-standard, if anything)

Reply:
we specify the port for metrictank, I think it's clearer and more consistent to have it here also just so there's no confusion.